build.gradle

```diff
@@ -53,14 +53,6 @@ allprojects {
         maven {
             url = "https://modmaven.dev/"
         }
-        maven {
-            name "aether"
-            url = "https://maven.pkg.github.com/Gilded-Games/The-Aether"
-            credentials {
-                username "$mavenDownUser"
-                password "$mavenDownPassword"
-            }
-        }
```

Comment:
Does Gilded Games no longer publish artifacts to this this maven?

Reply:
i don't think so, i just switched for simplicity and to avoid username/password, i'll rollback changes

Reply:
After further investigation it seems the project has moved to https://maven.pkg.github.com/The-Aether-Team/The-Aether (https://github.com/The-Aether-Team/The-Aether) adapted to use github maven

Reply:
No, I don't think so. IIrc the datapack loading stuff is broken, as such those custom portals aren't working. (You'd also need a datapack enabling compatibility)